src/prefect/runner/utils.py

```diff
@@ -0,0 +1,79 @@
+import typing as t
+
+from prefect._vendor.fastapi import FastAPI
+from prefect._vendor.fastapi.openapi.utils import get_openapi
+
+
+def _inject_schemas_into_generated_openapi(
+    webserver: FastAPI, schemas: t.Dict[str, t.Any]
+) -> t.Dict[str, t.Any]:
+    """Injects the parameter schemas into the Runner's Webserver OpenAPI schema
+
+    Args:
+        - webserver: the webserver to inject the schemas into
+        - schemas: the schemas to inject
+    """
+    openapi_schema = get_openapi(
+        title="FastAPI Prefect Runner", version="2.5.0", routes=webserver.routes
```

Comment:
is this version meaningful?

Reply:
hmm it doesn't appear to be so meaningful, I think we're just mirroring their `Info` object in our vendored implementation and elsewhere we appear to either use a hardcoded default or [pass our SDK version](https://github.com/PrefectHQ/prefect/blob/aed0ba764800797da2f7e778dbecb8d615e3f03c/src/prefect/server/api/server.py#L584). I prefer the latter here, in lieu of some other meaning, so I'll run with that if that sounds okay.

Reply:
using `prefect.__version__` as of [ead546e](https://github.com/PrefectHQ/prefect/pull/11333/commits/ead546e6f2f12556b0e72b98c64fb69e8725899f)

Reply:
Yea I agree that the SDK version is more meaningful considering how tightly this webserver integrates with the underlying flow code 👍